src/main/scala/akka/stream/contrib/Sample.scala

```diff
@@ -0,0 +1,64 @@
+package akka.stream.contrib
+
+import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
+import akka.stream.{Attributes, FlowShape, Inlet, Outlet}
+
+object Sample
+{
+  /**
+    *
+    * returns every nth elements
+    *
+    * @param nth
+    * @tparam T
+    * @return
+    */
+  def apply[T](nth: Int): Sample[T] = Sample[T](() => nth)
+
+  // def random[T](maxStep: Int = 1000, random: Random = Random): Sample[T] = Sample[T](() => random.nextInt(maxStep))
```

Comment:
instead of `scala.Random` I think it should be `java.util.Random`.

I think the default implementation should use `java.util.concurrent.ThreadLocalRandom`, i.e.

    def random[T](maxStep: Int = 1000): Sample[T] = Sample[T](() => ThreadLocalRandom.current().nextInt(maxStep))